packages/commonwealth/client/scripts/views/pages/create_community/index.tsx

```diff
@@ -132,7 +135,7 @@ const CreateCommunity = (props: CreateCommunityProps) => {
     };
 
     fetchEthChains();
-  }, []);
+  }, [setEthChains]);
```

Comment:
I think we can remove `setEthChains` from the dep array as React guarantees that state callbacks will keep references intact during component updates

Reply:
Again, Ifu added this here because of eslint warning. And yes, you are right @mzparacha that `setEthChains` is a setter that React guarantees to be unchanged over the rerenders. The problem here is though, that this setter is not defined in the component but it comes from the custom hook and eslint does not know about it and we can't do anything about it. The thing is that we are not loosing anything if we add those setters to the dependecy list and I would prefer if we would go this way down the road. It is for sure better than disabling eslint for this line. 

[Adding SO thread on this for more context](https://stackoverflow.com/questions/62419962/how-to-let-eslint-react-hooks-exhaustive-deps-rule-know-that-the-return-value-of).

Reply:
@mzparacha 
@masvelio is correct about the reasoning behind this. ESLint is quite strict about what goes in the dependencies array; asking even for things that don't actually change.